app/src/main/res/layout/scroll_view.xml

```diff
@@ -0,0 +1,336 @@
+        android:scaleY=".5"
+        android:stepSize="1"
+        android:textAppearance="@style/tag"
+        android:theme="@style/MyRatingBar"
+        android:transformPivotX="0dp"
+        android:transformPivotY="0dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/TextViewTegLine2" />
+
+    <TextView
+        android:id="@+id/textViewReview"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/textViewReview"
+        android:textAllCaps="true"
+        android:textColor="@color/colorTextGrey"
+        android:textSize="16sp"
+        app:layout_constraintEnd_toStartOf="@+id/guidelineCenter2"
+        app:layout_constraintStart_toStartOf="@+id/guidelineCenter2"
+        app:layout_constraintTop_toBottomOf="@+id/TextViewTegLine2" />
+
+    <TextView
+        android:id="@+id/textViewStoryLine"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        android:text="@string/textViewStoryLine"
+        android:textAppearance="@style/after_the_d"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/ratingBar" />
+
+    <TextView
+        android:id="@+id/StyleStoryLineContent"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+
+        android:layout_marginEnd="16dp"
+        android:text="@string/storyLineContent"
+        android:textAppearance="@style/after_the_d"
+
+        app:layout_constraintTop_toBottomOf="@+id/textViewStoryLine"
+        tools:layout_editor_absoluteX="16dp" />
+
+
+    <TextView
+        android:id="@+id/heading"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="20dp"
+        android:gravity="center_vertical"
+
+        android:lineSpacingExtra="8sp"
+        android:text="@string/heading"
+        android:textAppearance="@style/heading"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/StyleStoryLineContent" />
+
+
+    <View
+        android:id="@+id/movie1"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="4dp"
+        android:layout_marginEnd="4dp"
+        android:layout_marginBottom="4dp"
+        android:background="@drawable/ovie"
+        app:layout_constraintDimensionRatio="1:1"
+        app:layout_constraintEnd_toStartOf="@id/movie2"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/heading"
+        tools:ignore="ObsoleteLayoutParam" />
+
+    <TextView
+        android:id="@+id/textView1"
```

Comment:
@kamar19 Please avoid such ids naming as `textView1` Please add the name depends on the context of using this view.

Reply:
Accepted. Correcting.